Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -4310,21 +3007,31 @@ script:
     - contextPath: PaloAltoNetworksXDR.FeaturedField.fields
       description: String value that defines the new field.
       type: String
-  dockerimage: demisto/python3:3.10.8.36650
-  feed: false
+  - arguments:
+    - description: List of endpoint IDs. Supports comma separated list.
+      isArray: true
+      name: endpoint_ids
+    - description: Tag to add.
+      name: tag
+    description: Adds a tag to specified endpoint_ids
+    name: xdr-endpoint-tag-add
+  - arguments:
+    - description: List of endpoint IDs. Supports comma separated list.
+      isArray: true
+      name: endpoint_ids
+    - description: Tag to remove from specified endpoint_ids
+      name: tag
+    description: Removes a tag from specified endpoint_ids.
+    name: xdr-endpoint-tag-remove
```

Comment:
I think the all the args in  this case should be mandatory, please add-
`required: true` to all of them.

Reply:
Agree.  Added in latest commit.

Reply:
As for the core tagging commands.  I see that they were supposedly added [here](https://github.com/demisto/content/blob/ffa51c37c5c9c2c706a8f55eb827268100c04438/Packs/CortexXDR/ReleaseNotes/4_8_14.md) but looking at the most recent release the commands simply aren't there and the release notes for 4.8.14 are empty when viewed on the marketplace.

Also the CortexCoreIR is under the Core? pack focused on XSIAM and this is the XDR pack.  Thoughts?